### ISSUE TITLE
Send Amplitude events from the OAuth server

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/fxa#802]
In https://github.com/mozilla/fxa-auth-server/pull/3007, we started sending events to Amplitude from the OAuth server. I naively expected events to magically appear in Amplitude w/o consulting @jbuck or @jrgm whether any additional work is needed. Turns out, yes, additional work is needed.

This is a placeholder to track getting amplitude events from the OAuth server to amplitude.

IIUC, the general plan is:

1. Send OAuth server events to Stackdriver (already done)
2. Export applicable events from Stackdriver to Amplitude (not done)

@jbuck is working on step 2. @jbuck can you add context that observers would find useful?

blocks #795

**Comment by shane-tomlinson** [comment]
Blocked by https://github.com/mozilla/fxa-amplitude-send/issues/89

**Comment by jbuck** [comment]
This is deployed and running in stage/prod as of now!

🎊

**Comment by shane-tomlinson** [comment]
> This is deployed and running in stage/prod as of now!
>
> 🎊

This is *amazing*!

**Comment by shane-tomlinson** [comment]
From Amplitude:

<img width="1125" alt="Screenshot 2019-06-03 at 10 12 34" src="https://user-images.githubusercontent.com/848085/58790347-20295f80-85e8-11e9-8080-a099cef0cfe1.png">


Marking as DONE! Thanks @jbuck and @philbooth!
